src/features/home/Home.js

```diff
@@ -122,16 +123,38 @@ export default function Home() {
                   <li data-monthly-total="budgets">
                     <BsCalendar className="mr-1" />
                     <strong>Total dos orçamentos: </strong>
-                    <span className={monthlyBudgetCalcs.total.lessThan(0) ? 'text-danger' : ''}>
-                      R$ {monthlyBudgetCalcs.total.toFixed(2)}
-                    </span>
+                    <NumberFormat
+                      value={Number(monthlyBudgetCalcs.total)}
+                      defaultValue={Number(monthlyBudgetCalcs.total)}
+                      displayType="text"
+                      fixedDecimalScale
+                      decimalSeparator={','}
+                      thousandSeparator={'.'}
+                      decimalScale={2}
+                      renderText={value => (
```

Comment:
I think you could do here the same thing I mentioned on the following comments. What do you think?